packages/block-library/src/navigation/edit.js

```diff
@@ -71,18 +71,18 @@ function Navigation( {
 				return null;
 			}
 
-			return pages.map( ( { title, type, link: url, id } ) => (
-				createBlock( 'core/navigation-link',
+			return pages.map( ( { title, type, link: url, id } ) => {
+				return createBlock( 'core/navigation-link',
```

Comment:
What's the benefit of explicitly returning? Is that a style preference in Gutenberg?

Reply:
nah, I forgot that there!

Reply:
thanks for the notice! :)